File: captainhcg/go/26-remove-duplicates-from-sorted-array.go
func removeDuplicates(nums []int) int {
    if len(nums) == 0{
        return 0
    }
    st, ed := 1, 1
    for ed < len(nums) {
        if nums[ed] != nums[ed-1] {
            nums[st] = nums[ed]
            st += 1
        }
        ed += 1
    }
    return st
}

File: bwu/array/26-remove-duplicates-from-sorted-array.py
class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        p, n = 1, len(nums)
        if n == 0:
            return 0
            
        for i in xrange(1, n):
            if nums[i] != nums[i - 1]:
                nums[p] = nums[i]
                p += 1
        return p
